doc/source/io.rst

```diff
@@ -137,8 +137,9 @@ usecols : array-like or callable, default ``None``
 
   Using this parameter results in much faster parsing time and lower memory usage.
 as_recarray : boolean, default ``False``
-  DEPRECATED: this argument will be removed in a future version. Please call
-  ``pd.read_csv(...).to_records()`` instead.
+  .. deprecated:: 0.18.2
+     This argument will be removed in a future version. Please call
```

Comment:
I would eliminate things like: ``This argument will be removed in a future version.`` (look for occurences in the changes you made)

as the deprecated tag already has this (I think)